content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -222,6 +222,25 @@ This is an informational extension point. Post-bind plugins are called after a
 Pod is successfully bound. This is the end of a binding cycle, and can be used
 to clean up associated resources.
 
+### EnqueueExtension
+
+EnqueueExtension is the interface where the plugin can have control how the scheduling of Pods rejected by this plugin are retried.
```

Comment:
How about 

```suggestion
EnqueueExtension is the interface where the plugin can control the retry behavior
when the Pods is rejected.
```

Reply:
Well, this suggestion would lose one important thing -- it controls the retry of the scheduling of Pods **rejected by this plugin are retried.**
Any other good idea to make it better?

Reply:
```suggestion
EnqueueExtension is the interface where the plugin can control whether to retry scheduling of Pods rejected by this plugin, based on changes in the cluster.
```